weblate/static/editor/browse.js

```diff
@@ -0,0 +1,28 @@
+(() => {
```

Comment:
Copyright header missing, something like:

```suggestion
// Copyright © Michal Čihař <michal@weblate.org>
//
// SPDX-License-Identifier: GPL-3.0-or-later

(() => {
```

See https://github.com/WeblateOrg/weblate/actions/runs/10006505825/job/27659356277?pr=12093#step:7:47